postgres/init/21-portal-basedata.sql

```diff
@@ -478,6 +478,14 @@ COPY portal.notification_type (id, label) FROM stdin;
 \.
 
 
+--
+-- Data for Name: notifications; Type: TABLE DATA; Schema: portal; Owner: portal
+--
+
+COPY portal.notifications (id, receiver_user_id, date_created, content, notification_type_id, is_read, due_date, creator_user_id) FROM stdin;
```

Comment:
notifications certainly belong to content, not basedata

Reply:
moved it.